app/models/middleware_server.rb

```diff
@@ -2,5 +2,6 @@ class MiddlewareServer < ActiveRecord::Base
   include ReportableMixin
 
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  has_many :middleware_deployments, :foreign_key => "server_id"
```

Comment:
@Fryguy, your thoughts on the topic? https://github.com/abonas/manageiq/pull/1#discussion_r47475549